packages/devtools/initializers/create/src/main.ts

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright 2023 DXOS.org
+//
+import minimist from 'minimist';
+
+import template from '@dxos/hello-template';
+
+void (async () => {
+  const args = minimist(process.argv.slice(2), {
+    boolean: ['interactive', 'verbose', 'monorepo']
+  });
+  const { monorepo, interactive, verbose } = { interactive: false, verbose: false, monorepo: false, ...args };
+  const files = await template.execute({
+    verbose,
+    input: interactive
+      ? { monorepo }
+      : { pwa: true, dxosUi: true, tailwind: true, react: true, monorepo, storybook: false }
+  });
+  await Promise.all(files.map((file) => file.save()));
```

Comment:
- What directory does this output in by default?
- Is there a way to change that?
- Does `npm install` automatically get run on that directory after this executes?
- How is the vite patch being dealt with in that case?

Reply:
`cwd` by default, no sign of any npm convention besides that, but there is argument forwarding which can be used to special case an argument or flag or something
npm install does not run after this, that's always a separate step (which I think should not be automated without some prompt or flag) and i think that's why the patch works fine